Set the 'planespeed' bit in patch features on.

From: Jan Hudec <bulb@ucw.cz>

The 'planespeed' bit in patch features (action 0x07 or 0x09 testing variable
0x85), so newgrf actually read the planespeed setting (patch variable 0x10).
Without that bit, sets ignore it, because TTDP without that bit would return
garbage.
---

 src/newgrf.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/newgrf.cpp b/src/newgrf.cpp
index d16bd06..0328f59 100644
--- a/src/newgrf.cpp
+++ b/src/newgrf.cpp
@@ -4207,7 +4207,7 @@ static void InitializeGRFSpecial()
 	                   |                                        (0 << 0x0C)  // newagerating
 	                   |       ((_patches.build_on_slopes ? 1 : 0) << 0x0D)  // buildonslopes
 	                   |         ((_patches.full_load_any ? 1 : 0) << 0x0E)  // fullloadany
-	                   |                                        (0 << 0x0F)  // planespeed
+	                   |                                        (1 << 0x0F)  // planespeed
 	                   |                                        (0 << 0x10)  // moreindustriesperclimate - obsolete
 	                   |                                        (0 << 0x11)  // moretoylandfeatures
 	                   |                                        (1 << 0x12)  // newstations
